Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -1476,7 +1477,7 @@ getQuotesPolling action retryAction count duration push state = do
       else do
         -- let tempA = (getValueToLocalStore GOT_ONE_QUOTE)
         let gotQuote = (getValueToLocalStore GOT_ONE_QUOTE)
-        let usableCount = if gotQuote == "TRUE" && count > 9 then 9 else count
+        let usableCount = if gotQuote == "TRUE" && count > (fromMaybe 0 (fromString (getValueToLocalStore TEST_MINIMUM_POLLING_COUNT))) then (fromMaybe 0 (fromString (getValueToLocalStore TEST_MINIMUM_POLLING_COUNT))) else count
```

Comment:
store this `(fromMaybe 0 (fromString (getValueToLocalStore TEST_MINIMUM_POLLING_COUNT)))` in a variable and use